playground/vanillajs/index.html

```diff
@@ -1,12 +1,13 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <script>
-      // 👇 PUT YOUR API KEY HERE 👇
-      window.__clerk_publishable_api = '';
-      // 👆 PUT YOUR API KEY HERE 👆
-    </script>
-    <script src="../../packages/clerk-js/dist/clerk.browser.js"></script>
+    <script
+      async
+      crossorigin="anonymous"
+      data-clerk-publishable-key="<YOUR_PUBLISHABLE_KEY_GOES_HERE>"
+      src="https://<YOUR_FRONTEND_API_URL_GOES_HERE>/npm/@clerk/clerk-js@latest/dist/clerk.browser.js"
```

Comment:
☁️ @gkats I think we should change the src to `https://js.lclclerk.com/npm/clerk.browser.js` or something else that will point to a local build of the ClerkJS in order to test the changes applied in ClerkJS in the local repo.
Most of the time, playground apps are used to test the local changes. Since this app is probably not used a lot, you can ignore my comment